lib/request_overrider.js

```diff
@@ -91,6 +76,14 @@ class InterceptedRequestRouter {
     req.flushHeaders = (...args) => this.handleFlushHeaders(...args)
     req.abort = (...args) => this.handleAbort(...args)
 
+    // https://github.com/nock/nock/issues/256
+    if (options.headers.expect === '100-continue') {
```

Comment:
This event is being emitted a little bit lower in this function now. (Before it happened in the `for` loop.)